uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Test/Fabricator/AbsenceType.php

```diff
@@ -22,4 +22,27 @@ public static function fabricate($params = []) {
     return AbsenceType::create($params);
   }
 
+  /**
+   * Since we cannot create reserved types through the API,
+   * we have this helper method to insert one directly in
+   * the database
+   */
+  public static function createReservedType() {
+    $title = 'Title ' . microtime();
```

Comment:
This new method is a bit inconsistent with the other fabricate methods:
- it doesn't start with `fabricate`
- it doesn't accept a params array

My suggesting to fix this:
- rename it to fabricateReservedType
- add a $params param to it
- Internally, it will call fabricate($params) to create the absence type. Next, it will execute an UPDATE query to set is_reserved to 1

What do you think?